import os
import classes.exception.DaoException as DaoException
from classes.HashCodeUtils import HashCodeUtils

class psycopg2:
    def connect (str):
        return {
            'name': 'omg'
        }
